### ISSUE TITLE
Wazuh Kibana app doesn't install

### ISSUE THREAD

**Issue by mgmacias95** [sonofagl1tch/AWSDetonationLab#2]
Hi @sonofagl1tch,

After the deployment of my Wazuh manager in the Detonation lab, the Wazuh app wasn't installed. I run the script manually and noticed the following error:

```
Attempting to transfer from https://packages.wazuh.com/wazuhapp/wazuhapp-3.6.1.0_6.4.0.zip
Attempting to transfer from https://artifacts.elastic.co/downloads/kibana-plugins/https://packages.wazuh.com/wazuhapp/wazuhapp-3.6.1.0_6.4.0.zip/https://packages.wazuh.com/wazuhapp/wazuhapp-3.6.1.0_6.4.0.zip-6.4.0.zip
Plugin installation was unsuccessful due to error "No valid url specified."
```

I think the problem is on this part of the script:
https://github.com/sonofagl1tch/AWSDetonationLab/blob/419be40be284e0227bea699da0f4b02f07392e4d/additionalInstallationScripts/installWazuh.sh#L108-L109

More precisely, the Wazuh version variable is 3.6.1 but in the URL it is hardcoded a `.0`, resulting in a file `wazuhapp-3.6.1.0` which doesn't exist.
https://github.com/sonofagl1tch/AWSDetonationLab/blob/419be40be284e0227bea699da0f4b02f07392e4d/additionalInstallationScripts/installWazuh.sh#L6-L7

Best regards,
Marta

**Comment by sonofagl1tch** [comment]
corrected versioning hardcoded in the installation script. Thanks for catching that!

**Comment by mgmacias95** [comment]
Hi @sonofagl1tch,

I'm sorry, the fix you did is not correct. The wazuh version must be 3.6 because if not, this line will fail:

https://github.com/sonofagl1tch/AWSDetonationLab/blob/419be40be284e0227bea699da0f4b02f07392e4d/additionalInstallationScripts/installWazuh.sh#L80

The correct URL is 3.6, not 3.6.1:

https://raw.githubusercontent.com/wazuh/wazuh/3.6/extensions/elasticsearch/wazuh-elastic6-template-alerts.json

Wazuh has a branch 3.6.1 right now and that's why its working, but that branch is outdated and probably this fix won't be usable once Wazuh v3.7 gets released.

I'd fix it defining an extra variable `WAZUH_PATCH_VERSION=1`. This way, the kibana app url would be:

```
/usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-$WAZUH_VERSION.$WAZUH_PATCH_VERSION_$ELASTIC_VERSION.zip 
``` 

Best regards,
Marta
